api-reference/10 UI Widgets/dxDataGrid/1 Configuration/export/export.md

```diff
@@ -7,10 +7,355 @@ type: Object
+            selector: 'app-root',
+            templateUrl: './app.component.html',
+            styleUrls: ['./app.component.css']
+        })
+        export class AppComponent {
+            // ...
+        }
+
+
+        <!-- tab: app.module.ts -->
+        import { BrowserModule } from '@angular/platform-browser';
+        import { NgModule } from '@angular/core';
+        import { AppComponent } from './app.component';
+
+        import { DxDataGridModule } from 'devextreme-angular';
+
+        @NgModule({
+            declarations: [
+                AppComponent
+            ],
+            imports: [
+                BrowserModule,
+                DxDataGridModule
+            ],
+            providers: [ ],
+            bootstrap: [AppComponent]
+        })
+        export class AppModule { }
+
+
+    ##### Vue
+
+        <!-- tab: App.vue -->
+        <template>
+            <DxDataGrid ... >
+                <DxExport
+                    :enabled="true"
+                />
+                <DxColumn ... 
+                    :allow-exporting="false"
+                />
+            </DxDataGrid>
+        </template>
+
+        <script>
+        import 'devextreme/dist/css/dx.common.css';
+        import 'devextreme/dist/css/dx.light.css';
+
+        import ExcelJS from 'exceljs';
+        import saveAs from 'file-saver';
+
+        import { DxDataGrid, 
+            DxExport,
+            DxColumn
+        } from 'devextreme-vue/data-grid';
+
+        export default {
+            components: {
+                DxDataGrid,
+                DxExport,
+                DxColumn
+            }
+        }
+        </script>
+
+    ##### React
+
+        <!-- tab: App.js -->
+        import React from 'react';
+        import 'devextreme/dist/css/dx.common.css';
+        import 'devextreme/dist/css/dx.light.css';
+
+        import ExcelJS from 'exceljs';
+        import saveAs from 'file-saver';
+
+        import DataGrid, {
+            Export,
+            Column
+        } from 'devextreme-react/data-grid';
+
+        class App extends React.Component {
+            render() {
+                return (
+                    <DataGrid ... >
+                        <Export enabled={true} />
+                        <Column ...
+                            allowExporting={false}
+                        />
+                    </DataGrid>
+                );
+            }
+        }
+        export default App;
+
+    ---
+
+3. **Specify the Export Options**   
+    Call the [exportDataGrid(options)](/Documentation/ApiReference/Common/Utils/excelExporter/#exportDataGridoptions) method in the [onExporting](https://js.devexpress.com/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#onExporting) handler to execute export after the Export button is pressed. The deprecated export is deactivated by setting the `e.cancel` parameter to **true**. To save the document to Excel, use the **saveAs** function from the **FileSaver** API.
```

Comment:
The order of things in this paragraph is unnatural. The correct order is the following:

implement the handler -> call exportDataGrid in it -> use ExcelJS API to customize the document in customizeCell (and we no longer need step 4) -> call saveAs -> disable the deprecated export implementation